workflows/theiameta/wf_theiameta_panel_illumina_pe.wdl

```diff
@@ -0,0 +1,195 @@
+version 1.0
+
+import "../../tasks/alignment/task_minimap2.wdl" as minimap2_task
+import "../../tasks/assembly/task_metaspades.wdl" as metaspades_task
+import "../../tasks/quality_control/basic_statistics/task_fastq_scan.wdl" as fastq_scan
+import "../../tasks/quality_control/basic_statistics/task_quast.wdl" as quast_task
+import "../../tasks/quality_control/read_filtering/task_pilon.wdl" as pilon_task
+import "../../tasks/task_versioning.wdl" as versioning
+import "../../tasks/taxon_id/contamination/task_kraken2.wdl" as kraken_task
+import "../../tasks/taxon_id/task_krakentools.wdl" as krakentools_task
+import "../../tasks/utilities/data_handling/task_gather_scatter.wdl" as gather_scatter_task
+import "../../tasks/utilities/data_handling/task_parse_mapping.wdl" as parse_mapping_task
+import "../utilities/wf_morgana_magic.wdl" as morgana_magic_workflow
+import "../utilities/wf_read_QC_trim_pe.wdl" as read_qc_trim_pe
+
+workflow theiameta_panel_illumina_pe {
+  input {
+    String samplename
+    File read1
+    File read2
+    # default taxon IDs for Illumina VSP panel
+    Array[Int] taxon_ids = [10244, 10255, 10298, 10359, 10376, 10632, 10804, 11021, 11029, 11033, 11034, 11036, 11039, 11041, 11053, 11060, 11069, 11070, 11072, 11079, 11080, 11082, 11083, 11084, 11089, 11137, 11234, 11292, 11520, 11552, 11577, 11580, 11587, 11588, 11676, 11709, 12092, 12475, 12538, 12542, 28875, 28876, 31631, 33743, 35305, 35511, 36427, 37124, 38766, 38767, 45270, 46839, 57482, 57483, 59301, 64286, 64320, 68887, 80935, 90961, 95341, 102793, 102796, 108098, 114727, 114729, 118655, 119210, 129875, 129951, 130308, 130309, 130310, 138948, 138949, 138950, 138951, 147711, 147712, 152219, 162145, 169173, 186538, 186539, 186540, 186541, 238817, 277944, 290028, 333278, 333760, 333761, 333762, 440266, 463676, 493803, 536079, 565995, 862909, 1003835, 1216928, 1221391, 1239565, 1239570, 1239573, 1277649, 1313215, 1330524, 1335626, 1348384, 1424613, 1452514, 1474807, 1497391, 1608084, 1618189, 1891764, 1891767, 1965344, 1980456, 2010960, 2169701, 2169991, 2560525, 2560602, 2697049, 2847089, 2901879, 2907957, 3052148, 3052223, 3052225, 3052230, 3052302, 3052307, 3052310, 3052314, 3052470, 3052477, 3052480, 3052489, 3052490, 3052493, 3052496, 3052499, 3052503, 3052505, 3052518, 10798, 11216, 1203539, 12730, 142786, 1803956, 208893, 2560526, 2849717, 3052303, 3052317, 3052484, 3052498, 746830, 746831, 943908]
```

Comment:
Are you referring to the part where this line looks like a comment? Not sure why GitHub is rendering it like that, since it's not. 

Or are you talking about removing this as a default? I'd be hesitant to not have this as default unless we have come up with an easy method for users of the panel to copy-paste this information

Reply:
You're right! I assumed it was commented out, as it was rendered as such. Silly GH! Definitely want to leave it in as the default!